### PR TITLE
pkg/daemon/drain: Clarify "cordon/uncordon" messages

### PR DESCRIPTION
**Pull request by wking**
So we're symmetric for both cordoning and uncording.

Including the node name in the message is not strictly necessary, because it is already in the event's referenced object.  But the message is pretty short otherwise, so repeating the node name inline seemed like it might be a good use of space.

### CODE REVIEW

**Comment by kikisdeliveryservice** [comment]
we don't need this, sinny already has an approved PR that is waiting to merge:
https://github.com/openshift/machine-config-operator/pull/2657

/hold

File: pkg/daemon/drain.go [hunk 1]
```diff
@@ -38,16 +45,19 @@ func (dn *Daemon) cordonOrUncordonNode(desired bool) error {
 		err := drain.RunCordonOrUncordon(dn.drainer, dn.node, desired)
 		if err != nil {
 			lastErr = err
-			glog.Infof("cordon/uncordon failed with: %v, retrying", err)
+			glog.Infof("%s failed with: %v, retrying", verb, err)
```
Comment (wking) [inline pkg/daemon/drain.go:48]: @kikisdeliveryservice , any interest in this portion of the PR, or should we just close this?

Comment (sinnykumari) [reply pkg/daemon/drain.go:48]: yep, it makes sense to keep this part.

Comment (wking) [reply pkg/daemon/drain.go:46]: Rerolled to keep this, but leave eventing alone, in 5e2ca0f6 -> f302d7e4.  The new commit message tries to motivate logging but not including events within `cordonOrUncordonNode`, but let me know if my attempt at that motivation doesn't make sense.

File: pkg/daemon/drain.go [hunk 2]
```diff
@@ -38,16 +43,18 @@ func (dn *Daemon) cordonOrUncordonNode(desired bool) error {
...
 	}); err != nil {
 		if err == wait.ErrWaitTimeout {
-			return errors.Wrapf(lastErr, "failed to cordon/uncordon node (%d tries): %v", backoff.Steps, err)
+			return errors.Wrapf(lastErr, "failed to %s node (%d tries): %v", verb, backoff.Steps, err)
 		}
-		return errors.Wrap(err, "failed to cordon/uncordon node")
+		return errors.Wrapf(err, "failed to %s node", verb)
 	}
+
+	dn.logSystem("Node has been successfully %s", verb)
```
Comment (sinnykumari) [inline pkg/daemon/drain.go:57]: For consistency, I would say let caller decide both logging and sending event.

Comment (wking) [reply pkg/daemon/drain.go]: ok, rebased onto master and dropped this on-success log change with f302d7e4 -> 342c0135.

File: pkg/daemon/drain.go [hunk 3]
```diff
@@ -28,6 +28,11 @@ func (dn *Daemon) drainRequired() bool {
 }
 
 func (dn *Daemon) cordonOrUncordonNode(desired bool) error {
+	verb := "cordoned"
```
Comment (kikisdeliveryservice) [inline pkg/daemon/drain.go:31]:
```suggestion
	verb := "cordon"
```

File: pkg/daemon/drain.go [hunk 4]
```diff
@@ -28,6 +28,11 @@ func (dn *Daemon) drainRequired() bool {
...
+	if !desired {
+		verb = "uncordoned"
```
Comment (kikisdeliveryservice) [inline pkg/daemon/drain.go:33]:
```suggestion
		verb = "uncordon"
```

**Review by kikisdeliveryservice — COMMENTED** [review]
I like this, just 1 change to match sentence.

File: pkg/daemon/drain.go (hunk 3 above)
Comment (wking) [reply pkg/daemon/drain.go]: Done with 342c0135 -> da38bbbd.

File: pkg/daemon/drain.go (hunk 4 above)
Comment (wking) [reply pkg/daemon/drain.go]: Done with 342c0135 -> da38bbbd.

**Review by kikisdeliveryservice — APPROVED** [review]
Thanks Trevor!

**Comment by kikisdeliveryservice** [comment]
are commands dead or am impatient?

**Comment by kikisdeliveryservice** [comment]
HAHAHA. Impatient it is.

**Pull request merged**